src/plugins/data_views/common/data_views/data_view_lazy.ts

```diff
@@ -170,6 +170,7 @@ export class DataViewLazy extends AbstractDataView {
       { type, script },
       {
         customLabel,
+        customDescription,
```

Comment:
I think this change makes sense but it should be paired with a test to verify its doing the right thing.